src/jump_input.jl

```diff
@@ -0,0 +1,223 @@
+        println(io, "N $(length(lower_variables))")
+        println(io, "M $(length(lower_constraints))")
+        for x in lower_variables
+            println(io, "LC $(cols[x])")
+        end
+        for y in lower_constraints
+            println(io, "LR $(rows[y])")
+        end
+        for x in lower_variables
+            println(io, "LO $(obj_coefficients[x])")
+        end
+        println(io, "OS ", lower_sense == MOI.MAX_SENSE ? -1 : 1)
+    end
+    return
+end
+
+function _call_mibs(mps_filename, aux_filename)
+    io = IOBuffer()
+    MibS_jll.mibs() do exe
+        run(
+            pipeline(
+                `$(exe) -Alps_instance $(mps_filename) -MibS_auxiliaryInfoFile $(aux_filename)`,
+                stdout = io,
+            )
+        )
+    end
+    seekstart(io)
+    return read(io, String)
+end
+
+function _parse_output(
+    output::String,
+    new_model::MOI.FileFormats.MPS.Model,
+    lower_variables::Vector{MOI.VariableIndex}
+    )
+    lines = split(output, '\n')
+    found_status = false
+    objective_value = NaN
+
+    upper = Dict{Int,Float64}()
+    lower = Dict{Int,Float64}()
+
+
+    all_var = MOI.get(new_model, MOI.ListOfVariableIndices())
+
+    CntU = 0
+    CntD = 0
+
+    Dict_Lower_Name = Dict()
+    Dict_Lower_Value = Dict()
+    Dict_Upper_Name = Dict()
+    Dict_Upper_Value = Dict()
+
+    for (x, y) in MOI.enumerate(all_var)
+        nameofvar = MOI.get(new_model, MOI.VariableName(), y)
+        if y in lower_variables
+            Dict_Lower_Name[CntD] = nameofvar
+            Dict_Lower_Value[nameofvar] = 0
+            CntD = CntD + 1
+        else
+            Dict_Upper_Name[CntU] = nameofvar  
+            Dict_Upper_Value[nameofvar] = 0
+            CntU = CntU + 1
+        end
+    end
+
+
+
+    for line in lines
+        if !found_status
+            if occursin("Optimal solution", line)
+                found_status = true
+            end
+            continue
+        end
+        m = match(r"([xy])\[([0-9]+)\] \= (.+)", line)
+        if m === nothing
+            m = match(r"Cost \= (.+)", line)
+            if m !== nothing
+                objective_value = parse(Float64, m[1])
+            end
+            continue
+        end
+
+        column = parse(Int, m[2])
+        value = parse(Float64, m[3])
+
+        if m[1] == "x"
+            upper[column] = value
+            nameofvar = Dict_Upper_Name[column]
+            Dict_Upper_Value[nameofvar] = value
+        else
+            lower[column] = value
+            nameofvar = Dict_Lower_Name[column]
+            Dict_Lower_Value[nameofvar] = value
+        end
+    end
+
+    return (
+        status = found_status,
+        objective = objective_value,
+        nonzero_upper = upper,
+        nonzero_lower = lower,
+        all_upper = Dict_Upper_Value,
+        all_lower = Dict_Lower_Value
+    )
+end
+
+# input
+# fails on ,...
+# outputs
+
+function solve_with_MibS(model::BilevelModel; silent::Bool = true)
```

Comment:
```suggestion
"""

    solve_with_MibS(model::BilevelModel; silent::Bool = true)
    
Explain what the outputs are so that Joaquim can finish the rest.
"""
function solve_with_MibS(model::BilevelModel; silent::Bool = true)
```

Reply:
Example: https://github.com/odow/SDDP.jl/blob/3e14562b73db19b4267f3d42c113154093aa5b6e/src/algorithm.jl#L1269-L1285

Reply:
In the documentation outline the assumptions: i.e., that all variables are integer.